feature-assets/src/main/java/io/novafoundation/nova/feature_assets/domain/common/AssetPrefixNormalizing.kt

```diff
@@ -0,0 +1,26 @@
+package io.novafoundation.nova.feature_assets.domain.common
+
+import io.novafoundation.nova.common.utils.TokenSymbol
+import io.novafoundation.nova.common.utils.asTokenSymbol
+
+/**
+ * Some of tokens may have a prefix but may still be original token (like xcDOT, xcHDX)
+ * For some cases we need to normalize token symbol to original format
+ */
+
+private val normalizingPrefixList: List<String>
+    get() = listOf("xc")
+
+fun normalizeTokenSymbol(tokenSymbol: TokenSymbol): TokenSymbol {
+    return normalizeTokenSymbol(tokenSymbol.value).asTokenSymbol()
+}
```

Comment:
We can make it an extension function property for `TokenSymbol` to not to pollute "global namespace"